CHANGELOG/CHANGELOG-1.4.md

```diff
@@ -0,0 +1,88 @@
+- Add more visibility metrics to report the connection status of the Antrea Agent to the Flow Aggregator. ([#2668](https://github.com/antrea-io/antrea/pull/2668), [@zyiou])
+- Add the `antreaProxy.skipServices` configuration parameter for the Antrea Agent to specify Services which should be ignored by AntreaProxy. ([#2882](https://github.com/antrea-io/antrea/pull/2882), [@luolanzone])
+  * A typical use case is setting `antreaProxy.skipServices` to `["kube-system/kube-dns"]` to make [NodeLocal DNSCache] work when AntreaProxy is enabled.
+- Add support for `ToServices` in the rules of Antrea-native policies to allow matching traffic intended for Services. ([#2755](https://github.com/antrea-io/antrea/pull/2755), [@GraysonWu])
+- Add the `egress.exceptCIDRs` configuration parameter for the Antrea Agent, to specify IP destinations for which SNAT should not be performed on outgoing traffic. ([#2749](https://github.com/antrea-io/antrea/pull/2749), [@leonstack])
+- Add user documentation for [WireGuard encryption]. ([#2902](https://github.com/antrea-io/antrea/pull/2902), [@jianjuns])
+- Add user documentation for [encap mode installation for EKS]. ([#2929](https://github.com/antrea-io/antrea/pull/2929), [@jianjuns])
+
+
+### Changed
+
+- Remove chmod for OVSDB file from start_ovs, as the permissions are set correctly by OVS 2.15.1. ([#2803](https://github.com/antrea-io/antrea/pull/2803), [@antoninbas])
+- Reduce memory usage of antctl when collecting supportbundle. ([#2813](https://github.com/antrea-io/antrea/pull/2813), [@tnqn])
+- Do not perform SNAT for egress traffic to Kubernetes Node IPs. ([#2762](https://github.com/antrea-io/antrea/pull/2762), [@leonstack])
+- Send gratuitous ARP for EgressIP via the transport interface, as opposed to the interface with Node IP (if they are different). ([#2845](https://github.com/antrea-io/antrea/pull/2845), [@Jexf])
+- Ignore hostNetwork Pods selected by Egress, as they are not supported. ([#2851](https://github.com/antrea-io/antrea/pull/2851), [@Jexf])
+- Avoid duplicate processing of Egress. ([#2884](https://github.com/antrea-io/antrea/pull/2884), [@Jexf])
+- Ignore the IPs of kube-ipvs0 for Egress as they cannot be used for SNAT. ([#2930](https://github.com/antrea-io/antrea/pull/2930), [@Jexf])
+- Change flow exporter export expiry mechanism to priority queue based, to reduce CPU usage and memory footprint. ([#2360](https://github.com/antrea-io/antrea/pull/2360), [@heanlan])
+- Make Pod labels optional in the flow records. By default, they will not be included in the flow records. Use the `recordContents.podLabels` configuration parameter for the Flow Aggregator to include them. ([#2739](https://github.com/antrea-io/antrea/pull/2739), [@yanjunz97])
+- Wait for AntreaProxy to be ready before accessing Service if `antreaProxy.proxyAll` is enabled, to avoid connection issues on Agent startup. ([#2858](https://github.com/antrea-io/antrea/pull/2858), [@tnqn])
```

Comment:
If you mean a specific Service (e.g. the `antrea` Service) please mention it explicitly: "Wait for AntreaProxy to be ready before accessing the Antrea Service"

If you mean more than one service (e.g. `antrea` and `kubernetes`, please say: "Wait for AntreaProxy to be ready before accessing any K8s Service"

Reply:
done, thanks